lib/PuppeteerSharp/Page.cs

```diff
@@ -1315,6 +1318,17 @@ await Task.WhenAll(Frames.Select(frame => frame.EvaluateExpressionAsync(expressi
                 throw new NavigationException(response.errorText.ToString());
             }
         }
+
+        private static string EvaluationString(string fun, params object[] args)
```

Comment:
I tried to make the Helper class por specific, instead of a bag of helpers. I moved this method to the Page class, because it’s being used only there. And GetExceptionDetails to ExecutionContext, for the same reason.

After that, the remaining 2 methods in the Helper class were related to remote objects so I turned it into a RemoteObjectHelper.